OpenRA.Game/Graphics/Vertex.cs

```diff
@@ -46,4 +46,19 @@ public Vertex(float x, float y, float z, float s, float t, float u, float v, flo
 			R = r; G = g; B = b; A = a;
 		}
 	}
+
+	public sealed class CombinedShaderBindings : ShaderBindings
+	{
+		public CombinedShaderBindings()
+			: base("combined")
+		{ }
+
+		public override ShaderVertexAttribute[] Attributes { get; }	= new[]
+		{
+			new ShaderVertexAttribute("aVertexPosition", 0, 3, 0),
```

Comment:
Could the index be taken from the ordering of this `Attributes` array instead of explicitly defining them in the ctor?

Reply:
maybe. This was the original implementation of #19519. I would have to ask @IceReaper

Reply:
using array index as index now